packages/create-sitecore-jss/src/bin.test.ts

```diff
@@ -321,47 +304,20 @@ describe('bin', () => {
         const testArgs = mockArgs({
           destination: undefined,
         });
-        await getDestinations(testArgs, testTemplates);
+        await getDestination(testArgs, testTemplates);
         expect(inquirerPromptStub).to.have.been.calledOnce;
         expect(inquirerPromptStub.getCall(0).args[0].message).to.be.equal(
           'Where would you like your new app created?'
         );
       });
 
-      it('should prompt for both base and proxy when destinations are missing in args and templates contain proxy app', async () => {
```

Comment:
why are we removing all these tests? Should they be adjusted or moved?

Reply:
bin.ts will no longer prompt and set proxyAppDestination - that has been moved to angular-xmcloud initializer. This is done for consistency (initializer is called in both yarn watch and CLI command flows) and to focus on angular specifically for now. 
cross-framework possibility to install app+proxy is postponed for now, but we will get back to it when we have more resource to test all the frameworks and all the scenarios